app/src/main/java/org/mozilla/fenix/settings/TabsSettingsFragment.kt

```diff
@@ -69,9 +71,15 @@ class TabsSettingsFragment : PreferenceFragmentCompat() {
         listRadioButton = requirePreference(R.string.pref_key_tab_view_list_do_not_use)
         gridRadioButton = requirePreference(R.string.pref_key_tab_view_grid)
         searchTermTabGroups = requirePreference<SwitchPreference>(R.string.pref_key_search_term_tab_groups).also {
-            it.isVisible = FeatureFlags.tabGroupFeature
-            it.isChecked = it.context.settings().searchTermTabGroupsAreEnabled
+            it.isVisible = it.context.settings().showSearchGroupsFeature
+            it.isChecked = it.context.settings().showSearchGroupsFeature
             it.onPreferenceChangeListener = SharedPreferenceUpdater()
+
+            if (it.context.settings().showSearchGroupsFeature) {
+                requireContext().components.analytics.experiments.recordExposureEvent(
+                    FeatureId.SEARCH_TERM_GROUPS
+                )
+            }
```

Comment:
@jhugman @travis79 I'm not sure if having this section conditionally it's OK?

Reply:
James is correct, the "exposure" event is intended to record the point in the application flow where the user encounters the trigger to either display the "placebo" or control experience, or the treatment experience. We want to know when this happens regardless of which branch the user is enrolled in and Nimbus will take care of the rest.